### PR TITLE
Created help article for How-to add and manage attachments

### PR DESCRIPTION
**Pull request by anortiz08**

### CODE REVIEW

File: _articles/miscellaneous/attachments.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: article
+title: How-to attach files to an entry
```
Comment (kspearrin) [inline _articles/miscellaneous/attachments.md]: we call them "items"

File: _articles/miscellaneous/attachments.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+**Premium** accounts and organations include 1GB of storage and additional stoage can be purchased in 1GB increments via the account **Settings** or the organziation **Billing and Licesning** pages.
+
+## Attaching Files
+
+1. Create any entry type (login, identity, note, or card) and save it
+2. Once the entry has been saved, select the gear icon ![alt text](/images/icons/gear.png) next to the entry and select **Attachments**. You can attach any file 100MB or less.
+
+## Managing Attachments
+
+1. Select the gear icon ![alt text](/images/icons/gear.png) next to the entry you want to edit and select **Attachments**. You will then be presented with a list of all attached files for that entry
```
Comment (kspearrin) [inline _articles/miscellaneous/attachments.md]: No need for the image. Just use the icon tag. Ex: `{% icon fa-cog %}`

File: _articles/miscellaneous/attachments.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+layout: article
+title: How-to attach files to an entry
+categories: [miscellaneous]
+featured: false
+popular: false
+tags: [attachments, entry, storage, premium]
+---
+## Overview
+*Note: Support for attachments is a **PREMIUM ONLY** feature.*
```
Comment (kspearrin) [inline _articles/miscellaneous/attachments.md]: This should go above "Overview" and can use the notes tag for proper formatting:

```
{% note %}
Support for attachments is a **premium** feature.
{% endnote %}
```

File: _articles/miscellaneous/attachments.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+Attaching and managing files for an entry can be done easily in the bitwarden Web Vault. Any file type 100MB or smaller can be attached to an any of the four entry types (login, identity, note, or card).
```
Comment (kspearrin) [inline _articles/miscellaneous/attachments.md]: entry => "item in your vault". entry types => item types

File: _articles/miscellaneous/attachments.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+**Premium** accounts and organations include 1GB of storage and additional stoage can be purchased in 1GB increments via the account **Settings** or the organziation **Billing and Licesning** pages.
```
Comment (kspearrin) [inline _articles/miscellaneous/attachments.md]: organizations misspelled

**Comment by anortiz08** [comment]
@kspearrin Requested changes made, please review latest commit.

**Comment by kspearrin** [comment]
Can you delete the image?

**Comment by anortiz08** [comment]
@kspearrin Image deleted.

**Pull request merged**